src/helm/proxy/models.py

```diff
@@ -374,6 +374,31 @@ def engine(self) -> str:
         "JavaScript subset of The Stack (v1.1).",
         tags=[CODE_MODEL_TAG],
     ),
+    # Local HuggingFace models
+    Model(
+        group="local",
+        creator_organization="Meta",
+        name="local/llama-7b",
+        display_name="LLaMA (7B)",
+        description="", # TODO add description
```

Comment:
Add description (you can use a snippet from the blog post).

Reply:
done!